docs/index.bs

```diff
@@ -1084,7 +1106,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
             1. Let |destination| be the {{ServiceWorkerContainer}} object whose associated [=ServiceWorkerContainer/service worker client=] is |targetClient|.
             1. Add a [=task=] that runs the following steps to |destination|'s [=ServiceWorkerContainer/client message queue=]:
                 1. Let |origin| be the [=Unicode serialization of an origin|Unicode serialization=] of |sourceSettings|'s [=environment settings object/origin=].
-                1. Let |source| be a {{ServiceWorker}} object, which represents the [=ServiceWorkerGlobalScope/service worker=] associated with |sourceSettings|'s [=environment settings object/global object=].
+                1. Let |source| be the result of [=getting the service worker object=] that represents |sourceSettings|'s [=environment settings object/global object=]'s [=ServiceWorkerGlobalScope/service worker=] in |targetClient|.
```

Comment:
I'm still wrapping my head around Realms, settings objects, etc, so a naive question: is there a more direct way to get to the service worker? It looks like we're doing [=context object=]'s [=relevant settings object=]'s [=global object=]'s service worker. Is it equivalent to do [=context object=]'s [=relevant global object=]'s service worker, or is that something else?

Reply:
Yeah, that's fair, I'll switch to that. I think it means adding `1. Let |contextObject| be the [=context object=].` at the start, since I don't think you can access the context object once you're in parallel.